libraries/AC_PrecLand/AC_PrecLand.h

```diff
@@ -87,6 +91,8 @@ class AC_PrecLand
         SITL = 4,
     };
 
+    void check_estimator_init_timeout();
```

Comment:
we should put a comment above this declaration of "check_estimator_init_timeout()" and also above the implementation in the .cpp file.